var/spack/repos/builtin/packages/amrex/package.py

```diff
@@ -107,7 +107,8 @@ class Amrex(CMakePackage, CudaPackage, ROCmPackage):
     # cmake @3.17: is necessary to handle cuda @11: correctly
     depends_on('cmake@3.17:', type='build', when='^cuda @11:')
```

Comment:
As discussed in https://github.com/AMReX-Codes/amrex/pull/2629:
Let's bump this with the minimal version we need for HIP
```suggestion
    depends_on('cmake@3.17:', type='build', when='^cuda @11:')
    # cmake @3.21: is necessary to handle HIP correctly
    depends_on('cmake@3.21:', type='build', when='+rocm')
```

Reply:
I was able to test it down to 3.20. I'll add the correction accordingly.